pkgs/games/crispy-doom/default.nix

```diff
@@ -11,6 +11,8 @@ stdenv.mkDerivation rec {
     sha256 = "1b6gn0dysv631jynh769whww9xcss1gms78sz3nrn855q1dsvcb4";
   };
 
+  patches = [ ./CVE-2020-14983.patch ];
```

Comment:
`fetchpatch` the upstream commits:

https://github.com/chocolate-doom/chocolate-doom/commit/f1a8d991aa8a14afcb605cf2f65cd15fda204c56.diff
https://github.com/chocolate-doom/chocolate-doom/commit/54fb12eeaa7d527defbe65e7e00e37d5feb7c597.diff

And add a comment to the first patch that it fixes CVE-2020-14983. This makes it easy to see where the patches come from.